.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+
+jobs:
+  build:
+    # We use machine executor as docker executor won't allow mounting of folders.
+    machine:
+      image: ubuntu-1604:201903-01 # Note: There is an overhead for provisioning a machine executor as a result of spinning
+      # up a private Docker server. Use of the machine key may require additional fees in a future pricing update.
+      docker_layer_caching: true
+
+    working_directory: ~/hypertrace
+    steps:
+      - checkout
+      - run: |
+          cd ./docker
+          docker-compose -f docker-compose.yml -f docker-compose-zipkin-example.yml up -d pinot || (docker-compose -f docker-compose.yml -f docker-compose-zipkin-example.yml logs; exit 1)
+          sleep 60
```

Comment:
It might be a complex to do here, but is there a way to get around wait with static time? The script could use condition with globally defined number of iterations and `sleep 1`.